plugin/cache/README.md

```diff
@@ -57,7 +57,12 @@ cache [TTL] [ZONES...] {
 * `serve_stale`, when serve\_stale is set, cache always will serve an expired entry to a client if there is one
   available.  When this happens, cache will attempt to refresh the cache entry after sending the expired cache
   entry to the client. The responses have a TTL of 0. **DURATION** is how far back to consider
-  stale responses as fresh. The default duration is 1h.
+  stale responses as fresh. The default duration is 1h. **REFRESH_MODE** controls when the attempt to refresh
+  the cache happens. `verified` will first verify that an entry is still unavailable from the source before sending
```

Comment:
A minor typo here, `verified` will cause a invalid value error

```suggestion
  the cache happens. `verify` will first verify that an entry is still unavailable from the source before sending
```

Reply:
@lobshunter, This PR merged a couple months ago, and the typo has since been fixed.